packages/wrangler/src/cfetch/index.ts

```diff
@@ -29,7 +29,7 @@ export async function fetchResult<ResponseType>(
   init: RequestInit = {},
   queryParams?: URLSearchParams
 ): Promise<ResponseType> {
-  const json = await fetchInternal<FetchResult<ResponseType>>(
+  const json = await fethInternal<FetchResult<ResponseType>>(
```

Comment:
????

Reply:
was trying to get tsc errors to show in the PR, I give up.